api/reports.py

```diff
@@ -172,6 +172,16 @@ def get_average_violations(camera_id):
     }
 
 
+@reports_api.get("/{camera_id}/no_face_mask_detections")
```

Comment:
The endpoint name sounds weird to me. What about `face_mask_stats` given that we return two values?

Reply:
Yes, I just used the name of the indicator in the fronted.
I changed now to your suggestion